File: parameters.java
import java.util.*; 
/*
Online Java - IDE, Code Editor, Compiler

Online Java is a quick and easy tool that helps you to build, compile, test your programs online.
*/


public class Parameters {
    public List<Float> itemWeights;
    public List<Float> itemValues;
    public Float maxWeightCapacity;
    
    public Parameters(List<Float> itemWeights, List<Float> itemValues, Float maxWeightCapacity) {
        this.itemWeights = itemWeights;
        this.itemValues = itemValues;
        this.maxWeightCapacity = maxWeightCapacity;
    }
    
    public static Parameters easy() {
        var weights = new ArrayList<Float>(){{
         add(1.2f);
         add(2.7f);
        }};
        
        var values = new ArrayList<Float>(){{
         add(4.7f);
         add(3.7f);
        }};
        return new Parameters(weights, values, 5f);
    }
    
	public static void main(String[] args) {
		System.out.println(easy().itemWeights);
	}
}

File: parameters.js
class Parameters {
    constructor(itemWeights, itemValues, maxWeightCapacity) {
        this.itemWeights = itemWeights;
        this.itemValues = itemValues;
        this.maxWeightCapacity = maxWeightCapacity;
    }

    static easy() {
        const values = [7250, 421, 7195, 49, 5077, 1537, 2229, 5653, 2366, 6881];
        const weights = [9818, 4040, 9560, 9436, 8080, 793, 886, 4294, 474, 2929];
        return new Parameters(weights, values, 5);
    }
}

console.log(Parameters.easy())